crates/allocator/src/bump.rs

```diff
@@ -264,6 +264,51 @@ mod tests {
         let expected_alloc_start = 2 * PAGE_SIZE + std::mem::size_of::<u8>();
         assert_eq!(inner.next, expected_alloc_start);
     }
+
+    // TODO: What I want to end up doing is turning this into a `quickcheck` test such that the
+    // random sized bytes and the number of allocations comes from `quickcheck`
```

Comment:
You could have a test a la
```
#[quickcheck]
fn can_alloc_many_different_sized_chunks(chunks: Vec<Vec<u8>>) {
  // …
}
```
…and the iterate over the `Vec` whilst allocating the inner chunk.